### PR TITLE
Refresh BigQuery Data Transfer

### PR DESCRIPTION
**Pull request by michaelbausor**
Command log:
```
g3artman --config third_party/googleapis/google/cloud/bigquery/datatransfer/artman_bigquerydatatransfer.yaml publish --target staging php_gapic
// Merge PR sent to api-client-staging repo
cd ~/workspace
git clone https://github.com/googleapis/api-client-staging.git
git clone https://github.com/GoogleCloudPlatform/google-cloud-php-bigquerydatatransfer.git
cd google-cloud-php-bigquerydatatransfer
cp -r ../api-client-staging/generated/php/google-cloud-bigquerydatatransfer-v1/src/BigQuery/DataTransfer/V1/* ./V1/
```

### CODE REVIEW

File: V1/DataTransferServiceClient.php [hunk 1]
```diff
@@ -1,12 +1,12 @@
 <?php
 /*
- * Copyright 2017, Google LLC All rights reserved.
+ * Copyright 2018 Google LLC
```
Comment (jdpedrie) [inline V1/DataTransferServiceClient.php]: Hmm, this should still be 2017.

File: V1/Gapic/DataTransferServiceGapicClient.php [hunk 2]
```diff
@@ -1,12 +1,12 @@
 <?php
 /*
- * Copyright 2017, Google LLC All rights reserved.
+ * Copyright 2018 Google LLC
```
Comment (jdpedrie) [inline V1/Gapic/DataTransferServiceGapicClient.php]: 2017

**Review by jdpedrie — COMMENTED** [review]
we could add this to the restructure too.

**Comment by michaelbausor** [comment]
Address comments. Re including in restructure PR - agreed, that makes sense. We should do whatever makes it easier to merge the restructure PR, so if we preferred to add this afterwards that would be fine too.

**Comment by theacodes** [comment]
Hey @michaelbausor, I've been making sure to log all commands used to generate the libraries (see [here](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/4877/commits/619cabf19a82b2d094d7d34a1590f9df1f1501c2)) can you do similar here?

**Comment by michaelbausor** [comment]
@jonparrott Updated comment, will include in commit message
@jdpedrie Addressed PR comments, PTAL

**Review by jdpedrie — APPROVED** [review]

**Pull request merged**